src/main/java/com/solvd/sql/services/CategoryService.java

```diff
@@ -1,7 +1,7 @@
 package com.solvd.sql.services;
 
 import com.solvd.sql.interfaces.ICategoryDao;
-import com.solvd.sql.jdbc.CategoryDao;
+import com.solvd.sql.mybatis.CategoryDao;
```

Comment:
Instead of replacing the import statement, we can initialize the service layer object by passing the DAO through the constructor. We declare the type as the entity interface since both JDBC and MyBatis DAOs implement from a common interface.